stdlib/public/stubs/Stubs.cpp

```diff
@@ -435,6 +440,287 @@ __mulodi4(di_int a, di_int b, int* overflow)
     }
     return result;
 }
+
+#if defined(_WIN64)
+
+#define CHAR_BIT      8         // number of bits in a char
```

Comment:
Can we not use the standard headers for this?

Reply:
Yes, sorry. I was blindly following Compiler-RT here, but it's already defined from `<limits.h>`.